### PR TITLE
rlimit constants update related to php 8.0.2 update

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Comment by cmb69** [comment]
FWIW, php/php-src#6608 has only been merged into the master branch, not PHP-8.0.

File: reference/posix/constants.xml [hunk 1]
```diff
@@ -358,6 +358,28 @@
...
+    </listitem>
+   </varlistentry>
+   <varlistentry xml:id="constant.posix-rlimit-npts">
+    <term>
+     <constant>POSIX_RLIMIT_NPTS</constant>
+      (<type>int</type>)
+    </term>
+    <listitem>
+     <simpara>
+      The maximum number of pseudo-terminals this user id is allowed to create (FreeBSD).
```
Comment (cmb69) [inline reference/posix/constants.xml:379]:
```suggestion
      The maximum number of pseudo-terminals this user id is allowed to create (FreeBSD).
      Available as of PHP 8.1.0.
```

File: reference/posix/constants.xml [hunk 2]
```diff
@@ -358,6 +358,28 @@
...
     </listitem>
    </varlistentry>
+   <varlistentry xml:id="constant.posix-rlimit-kqueues">
+    <term>
+     <constant>POSIX_RLIMIT_KQUEUES</constant>
+      (<type>int</type>)
+    </term>
+    <listitem>
+     <simpara>
+      The maximum number of kqueues this user id is allowed to create (FreeBSD).
```
[the remark this answers is not in the document]
Comment (devnexen) [reply reference/posix/constants.xml:368]: ah nevermind I requested PHP-8.0 at the time but nikic did only in master :-) ok will do.

**Review by cmb69 — APPROVED** [review]
Thanks!

**Pull request merged**